wsd/DocumentBroker.cpp

```diff
@@ -4064,8 +4070,7 @@ void DocumentBroker::uploadPresetsToWopiHost(const Authorization& auth)
     const std::string& jailPresetsPath = FileUtil::buildLocalPathToJail(
         COOLWSD::EnableMountNamespaces, getJailRoot(), JAILED_CONFIG_ROOT);
 
-    Poco::URI uriObject = DocumentBroker::getPresetUploadBaseUrl(_docKey);
-
+    Poco::URI uriObject = DocumentBroker::getPresetUploadBaseUrl(_uriOrig);
```

Comment:
Did we look at the differences between `_uriOrig` and `_uriPublic` to decide which one to use?

I would expect we want `_uriPublic`. Please read the code that sets these two values (hint: it's in `RequestVettingStation::handleRequest()`). Looking at the difference between them should make it clear why we want `_uriPublic`.

In fact, a quick grep shows that `_uriOrig` is only used in one case (argument to `COOLWSD::dumpNewSessionTrace()`), while `_uriPublic` is used _everywhere_ we need the URI of the document (grep for 'getPublicUri'). So, again, I'm not sure why we're not using the more sensible value. (We should check what we use and look at existing usage, otherwise ask when it's still not obvious.)

Reply:
Thanks